Makefile

```diff
@@ -1,12 +1,16 @@
 SHELL ?= /bin/bash
 
-ARGO_CD_CHART_VERSION := 5.46.6
+ARGO_CD_CHART_VERSION := 5.46.3
```

Comment:
Why did you change this?

Reply:
> Why did you change this?

trying to bootstrap the kind and the argocd repo server is crash-looping something with the Redis , so I tested it with a older version and for got to change it, sorry